src/database.ts

```diff
@@ -380,3 +380,72 @@ export async function getUserKarma(user: number): Promise<number> {
 
   return row?.karma || 0;
 }
+
+export async function getExlpsLikedByUser(
+  user: number,
+  limit: number | boolean = false,
```

Comment:
`null` should be used instead of boolean value, since `limit = true` does not make any sense in this context